src/test/java/picard/vcf/UpdateVcfSequenceDictionaryTest.java

```diff
@@ -54,51 +52,57 @@ public void teardown() {
     public static Object[][] outputFies() {
 
         return new Object[][] {
-                {OUTPUT_DATA_PATH + "updateVcfSequenceDictionaryTest-delete-me.vcf"},
+                {OUTPUT_DATA_PATH + "updateVcfSequenceDictionaryTest-delete-me"  + VcfUtils.COMPRESSED_VCF_ENDING},
+                {OUTPUT_DATA_PATH + "updateVcfSequenceDictionaryTest-delete-me" + VcfUtils.UNCOMPRESSED_VCF_ENDING},
                 {STD_OUT_NAME}
```

Comment:
That works for System.out but not for /dev/stdout. That's why https://github.com/broadinstitute/picard/pull/741 tried redirecting the output to FileDescriptor.